static/app/views/starfish/utils/sqlish/sqlish.pegjs

```diff
@@ -0,0 +1,33 @@
+Expression
+   = tokens:Token*
+
+Token
+   = Keyword / Parameter / CollapsedColumns / Whitespace / GenericToken
+
+Keyword
+  = Keyword:("SELECT"i / "INSERT"i / "DELETE"i / "FROM"i / "ON"i / "WHERE"i / "AND"i / "ORDER BY"i / "LIMIT"i / "GROUP BY"i / "OFFSET"i / JoinKeyword) {
```

Comment:
I notice there's ~40 keywords in the old parser, but only 11 here. Should the rest be added as well? Or are they no longer needed

Reply:
Great question! I ~should~ will make this more explicit soon. Right now I chose a subset because these are the keywords that get special treatment in formatting (newlines or bolding) so I "promoted" them. A better approach is to add all ~40 keywords, and have a special list to denote which ones are "special"